remediation_worker/jobs/azure_sql_sql_server_access_restricted_from_internet/README.md

```diff
@@ -0,0 +1,74 @@
+# Close Sql Server Port 1433 for a Network Security Group
+
+This job blocks public access to Sql server port 1433.
+
+### Applicable Rule
+
+#### Rule ID:
+5c8c26927a550e1fb6560cca
+
+#### Rule Name:
+The security group allows access to Sql server port (1433)
+
+## Getting Started
+
+### Prerequisites
+
+The provided Azure service principal must have the following permissions:
+`Microsoft.Network/networkSecurityGroups/read`
+`Microsoft.Network/networkSecurityGroups/write`
+
+A sample role with requisite permissions can be found [here](minimum_permissions.json)
+
+More information about already builtin roles and permissions can be found
+[here](https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles)
+
+### Running the script
+
+You may run this script using following commands:
+```shell script
+  pip install -r requirements.txt
+  python3 azure_sql_sql_server_access_restricted_from_internet.py
+```
+
+## Running the tests
+You may run test using following command under vss-remediation-worker-job-code-python directory:
+```shell script
+    pip install -r requirements-dev.txt
+    python3 -m pytest test
+```
+## Deployment
```

Comment:
Change Deployment info to the following-
Provision an instance by creating an Azure Virtual Machine to use for the worker. The minimum required specifications are 128 MB memory and 1/2 Core CPU.
Setup Docker on newly provisioned Azure Virtual Machine instance. You can refer to the [docs here](https://docs.microsoft.com/en-us/previous-versions/azure/virtual-machines/linux/docker-machine) for more information.
Deploy the worker docker image by SSH into the Azure Virtual Machine instance and run the following commands:
  ```shell script
  docker run --rm -it --name {worker_name}\
  -e VSS_CLIENT_ID={ENTER CLIENT ID}\
  -e VSS_CLIENT_SECRET={ENTER CLIENT SECRET}\
  -e AZURE_CLIENT_ID={ENTER AZURE_CLIENT_ID} \
  -e AZURE_CLIENT_SECRET={ENTER AZURE_CLIENT_SECRET} \
  -e AZURE_TENANT_ID={ENTER AZURE_TENANT_ID} \
  vmware/vss-remediation-worker:latest-python
  ```
This is the deployment info for Azure.